component/sections/cookies/variant_a.js

```diff
@@ -1,72 +1,48 @@
-import React, { useState } from "react";
+import React from "react";
+import { setCookie, getCookie } from "utils/cookies"
 
-function VariantA({ title, text, button1, button2 }) {
-  //to hide cookie component
-  const [showCookie, setShowCookie] = useState(true);
+
+function VariantA({ title, text, allowCookieBtn, denyCookieBtn }) {
+  const [showCookie, setShowCookie] = React.useState(() => {
+    const cookieExists = getCookie();
+    return cookieExists;
```

Comment:
This is unnecessary I guess having to save the value of `getCookie()` and returning it as a value for useState consumption. Just return with `getCookie()` should be suffice. Makes our code shorter and easier to understand.

Reply:
Sorry about this. I have updated the code. Thanks!